src/audio/tables/samplebank_table.c

```diff
@@ -0,0 +1,50 @@
+#include "attributes.h"
+#include "z64audio.h"
+
+// Symbol definition
+
+extern AudioTable gSampleBankTable;
+#pragma weak gSampleBankTable = sSampleBankTableHeader
+
+// Externs for table
+
+#define DEFINE_SAMPLE_BANK(name, medium, cachePolicy) \
+    extern u8 name##_Start[];                         \
+    extern u8 name##_Size[];
+#define DEFINE_SAMPLE_BANK_PTR(index, medium, cachePolicy) /*empty*/
+
+#include "assets/audio/samplebank_table.h"
+
+#undef DEFINE_SAMPLE_BANK
+#undef DEFINE_SAMPLE_BANK_PTR
+
+// Table header
+
+NO_REORDER AudioTableHeader sSampleBankTableHeader = {
```

Comment:
```suggestion
NO_REORDER const AudioTableHeader sSampleBankTableHeader = {
```
Even though IDO still doesn't put it in rodata, since it does end up in rodata I feel we should still mark these as const.

Reply:
Actually trying this and I get that `sSampleBankTableHeader` and `sSampleBankTableEntries` are reordered, though not sure why :( .

Reply:
Apparently it's because `const AudioTableHeader sSampleBankTableHeader` goes in .rodata while `const AudioTableEntry sSampleBankTableEntries[]` goes in .data when initially compiled. The linker script sorts .rodata after .data so it swaps them.